pyaedt/edb.py

```diff
@@ -1822,6 +1829,7 @@ def _create_cutout_multithread(
         custom_extent_units="mm",
         check_terminals=False,
         include_pingroups=True,
+        preserve_components_with_model=False,
     ):
         if is_ironpython:  # pragma: no cover
             self.logger.error("Method working only in Cpython")
```

Comment:
Error messages should be short but complete sentences with punctuation: This method works only in CPython.